code/modules/mob/living/living.dm

```diff
@@ -284,12 +284,13 @@
 
 	if(ismob(AM))
 		var/mob/M = AM
+		var/mob/living/carbon/human/H = M
```

Comment:
You define this, but you never use it.

Reply:
should be used at line 291,292 and 293 by ishuman(M). Unless I'm doing things wrong

Reply:
should it be var/mob/living/carbon/human/ instead of var/mob/living/carbon/human/H?

Reply:
Okay, I won't be condescending here. The issue is, you set the value of var/mob/living/carbon/human/H to M's value, but then you used M instead of your new H?

Reply:
If you were just going to check if that M was human in visible_message code, why cast it to a variable you don't use?

Reply:
If you were just going to check if that M was human in visible_message code, why cast it to a variable you don't use?

Reply:
because I'm dumb. Sorry I'll fix it